packages/gp2-components/src/templates/OutputDetailPage.tsx

```diff
@@ -67,32 +68,43 @@ const OutputDetailPage: React.FC<OutputDetailPageProps> = ({
   isAdministrator,
   ...output
 }: OutputDetailPageProps) => (
-  <div css={containerStyles}>
-    {isAdministrator ? (
-      <div css={buttonsContainer}>
-        <div css={leftButtons}>
-          <Link
-            noMargin
-            href={
-              gp2Routing.outputs({}).output({ outputId: output.id }).edit({}).$
-            }
-            buttonStyle
-            small
-            primary
+  <PageNotifications page="outputs">
```

Comment:
this page is not exactly `outputs`. for more clarity, maybe change it to "output" or "output-detail"?